tools/cellxgene_census_builder/src/cellxgene_census_builder/build_soma/manifest.py

```diff
@@ -54,7 +54,7 @@ def load_manifest_from_CxG() -> List[Dataset]:
     logging.info("Loading manifest from CELLxGENE data portal...")
 
     # Load all collections and extract dataset_id
-    datasets = fetch_json(f"{CXG_BASE_URI}curation/v1/datasets")
+    datasets = fetch_json(f"{CXG_BASE_URI}curation/v1/datasets?schema_version=3.0.0")
```

Comment:
There is a `CXG_SCHEMA_VERSION` global variable in globals.py - we should use that one.

Reply:
There's also CXG_SCHEMA_VERSION_IMPORT. I think it can make sense to have both (one to pin to a major/minor version if you want to do that eventually, one to list supported patch versions) but they should probably have more distinct names. Going to leave as-is for now.